### ISSUE TITLE
[Feature]: Detecting when there is corruption or missing image data/files

### ISSUE THREAD

**Issue by zatricky** [immich-app/immich#1003]
### Feature detail

Note that I'm still testing Immich, so I am not personally concerned about data loss ... yet. :-)

The NFS data store I was using for /usr/src/app/upload had some corruption issues and so became read-only. After restoring the NFS service some images have missing content due to that corruption (which is obviously not Immich's fault). See included screenshot.

If the original source device still has the content, it would be useful if Immich were able to detect the issue and then cause these images to be re-uploaded from that original source device.

![Screenshot_2022-11-21_15-54-48](https://user-images.githubusercontent.com/3970906/203169373-efa09a2b-d4bb-459f-a709-2971587b498d.png)

### Platform

Server

**Comment by alextran1502** [comment]
Can you help attach one of the corrupted files?

**Comment by zatricky** [comment]
[67e52940-075c-4cd2-b7df-857c0d7f5adf.webp.gz](https://github.com/immich-app/immich/files/10083025/67e52940-075c-4cd2-b7df-857c0d7f5adf.webp.gz)
Looking at the image binary it's just a whole lot of SOH's (01) repeated forever. The full image is also just 01 repeated - just that instead of 7592 bytes of SOH, it is 227 575 bytes. I'm not sure how to get the on-disk filename if you would like for me to get the file directly from the NFS (assuming that would help).

**Comment by alextran1502** [comment]
@zatricky No, I think the corrupted file there is good, so I can reproduce it when I work on the issue. 

Thank you for pinging me
